components/sections/ExportTranslations.tsx

```diff
@@ -75,17 +75,33 @@ const ExportTranslations = ({ translations, categoryName, triggerLabel }: Export
 
   const translSep = translationSeparator === TRANS_SEP_CUSTOM ? customTranslationSeparator : translationSeparator;
   const phraseSep = phraseSeparator === PHRASE_SEP_CUSTOM ? customPhraseSeparator : phraseSeparator;
-  const phrases = translations
+
+  const phrases = translations.categories
     .map(
       (translation) =>
-        translation.getTranslation(currentLanguage) +
-        (includeTranscriptions ? ` [${translation.getTranscription(currentLanguage)}]` : '') +
+        '[' +
+        translation.name.main +
+        ']' +
         translSep +
-        translation.getTranslation(otherLanguage) +
-        (includeTranscriptions ? ` [${translation.getTranscription(otherLanguage)}]` : '') +
-        phraseSep
+        '[' +
+        translation.name.source +
+        ']' +
+        '\n' +
+        translation.phrases.map((phraseId) => {
```

Comment:
Tady je ta chyba s čárkou, sčítáš tady řetězec a pole, protože výsledkem translation.phrases.map je pole. A JavaScript ti provede automatickou konverzi pole na řetězec (tj, spojí jeho položky a oddělí je čárkou). Musíš si to překonvertovat sám dle svého (možná přes .join ?)